api_spec.yaml

```diff
@@ -2711,7 +2717,12 @@ components:
       type: object
       description: should be a JSON object (not a JSON literal nor an array)
       example: {"test":123}
-      
+
+    grantPayload:
+      type: object
+      description: should be a JSON object (not a JSON literal nor an array)
+      example: { "email-verified": { v: true, t: 1645998687930 } }
```

Comment:
Will this be an arbitrary JSON from the db's point of view?

Reply:
yeah.